src/index.html

```diff
@@ -11,7 +11,128 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form>
+      <h1> FORM </h1>
+      <p> Будь ласка заповність форму</p>
```

Comment:
it would be better to avoid unnecessary spaces between HTML tags and text inside
```suggestion
      <h1>FORM</h1>
      <p>Будь ласка заповність форму</p>
```